MAUI/TabView/Getting-Started.md

```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
 
 ![Getting_Started](images/Getting_Started.png)
 
-N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
+
+## Populate ItemsSource
+
+Items can be added to the control using the `ItemsSource` property of [SfTabView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html).
+
+Using ItemsSource, objects of any class can be given as items for SfTabView. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content. 
```

Comment:
Objects of any class can be given as items for SfTabView by using ItemsSource,. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content.